src/ray/raylet/worker_pool.h

```diff
@@ -137,6 +137,12 @@ class WorkerPool : public WorkerPoolInterface, public IOWorkerPoolInterface {
   /// \return Void.
   void HandleJobFinished(const JobID &job_id);
 
+  /// \brief Check get job config by the specified job id.
+  ///
+  /// \param job_id ID of the job.
+  /// \return Job config if given job is running, else nullptr.
+  const rpc::JobConfig *GetJobConfig(const JobID &job_id) const;
```

Comment:
is it possible to return `optional<const rpc::JobConfig &>`? Or maybe we should store `shared_ptr`s in `all_jobs_`.

Reply:
Okey, I will change it.